src/shadowbox/server/server_access_key.ts

```diff
@@ -33,10 +34,11 @@ interface AccessKeyConfig {
 // The configuration file format as json.
 export interface AccessKeyConfigJson {
   accessKeys?: AccessKeyConfig[];
-  // The port number to use for new access keys.
-  defaultPort?: number;
   // Next AccessKeyId to use.
   nextId?: number;
+
+  // DEPRECATED: Use ServerConfigJson.defaultAccessKeyPort instead.
+  defaultPort?: number;
```

Comment:
If this has only been set on a couple of canary servers, can you think of a way to remove it altogether? It significantly complicates `getDefaultPort`, below.

Reply:
We told ASL19 and others on Reddit and Github about the single port option, so they may have the old field set.

My suggestion is to now break compatibility for now, and delete after a month or so, since we automatically migrate the servers to the new field.

Reply:
OK, sounds good.